cmd/azure-config-credentials-injector/main.go

```diff
@@ -91,6 +94,16 @@ func readCloudConfig(path string) (map[string]interface{}, error) {
 func prepareCloudConfig(cloudConfig map[string]interface{}, clientId string, clientSecret string) ([]byte, error) {
 	cloudConfig[clientIDCloudConfigKey] = clientId
 	cloudConfig[clientSecretCloudConfigKey] = clientSecret
+	if value, found := cloudConfig[useManagedIdentityExtensionConfigKey]; found {
+		if injectorOpts.disableIdentityExtensionAuth {
+			fmt.Printf("%s cleared\n", useManagedIdentityExtensionConfigKey)
+			cloudConfig[useManagedIdentityExtensionConfigKey] = false
+		} else {
+			if value == true {
+				fmt.Printf("Warning: %s is set to \"true\", injected credentials may not be used\n", useManagedIdentityExtensionConfigKey)
```

Comment:
Likewise, is this meant as a log? Do we not have any logging libraries initialised here?

Reply:
Both messages are indeed logs, I added `klog/v2` as the logger to the binary. It prints something like this now:

```
W0208 11:17:12.025252    7724 main.go:104] Warning: useManagedIdentityExtension is set to "true", injected credentials may not be used
```